config/config.go

```diff
@@ -429,7 +429,7 @@ func (rc *ReporterConfig) NewReporter(
 func (rc *ReporterConfig) newTransport(logger jaeger.Logger) (jaeger.Transport, error) {
 	switch {
 	case rc.CollectorEndpoint != "":
-		httpOptions := []transport.HTTPOption{transport.HTTPBatchSize(1), transport.HTTPHeaders(rc.HTTPHeaders)}
```

Comment:
@danehan any idea why your original PR #323 set this to 1?

Reply:
Given @danehan's silence on GitHub the last year, would you consider moving forward with this after a week of no response?